implementations/rust/ockam/ockam_core/src/routing/message/local_message.rs

```diff
@@ -209,53 +209,12 @@ impl LocalMessage {
 
         cfg_if! {
             if #[cfg(feature = "std")] {
-                // make sure to pass the latest tracing context
-                let new_tracing_context = Self::start_new_tracing_context(self.tracing_context.update(), "TransportMessage");
-                transport_message.with_tracing_context(new_tracing_context)
+                transport_message.with_tracing_context(self.tracing_context.to_string())
             } else {
                 transport_message
             }
         }
     }
-
-    /// - A new trace is started
-    /// - The previous trace and the new trace are linked together
-    ///
-    /// We start a new trace here in order to make sure that each transport message is always
-    /// associated to a globally unique trace id and then cannot be correlated with another transport
-    /// message that would leave the same node for example.
-    ///
-    /// We can still navigate the two created traces as one thanks to their link.
-    #[cfg(feature = "std")]
-    pub fn start_new_tracing_context(
```

Comment:
What are the implications of not using this anymore? Does this mean that we now have a single trace?

Reply:
Yes, we have now a single trace per message, instead of having one trace per node and having links between those traces (because that would be harder to query with SQL)